packages/smart-contracts/scripts-create2/contract-setup/setups.ts

```diff
@@ -0,0 +1,157 @@
+import { HardhatRuntimeEnvironmentExtended } from '../types';
+import { erc20SwapConversionArtifact } from '../../src/lib';
+import { batchPaymentsArtifact } from '../../src/lib';
+import utils from '@requestnetwork/utils';
+import {
+  updateBatchPaymentFees,
+  updatePaymentErc20FeeProxy,
+  updatePaymentEthFeeProxy,
+  updateChainlinkConversionPath,
+  updateRequestSwapFees,
+  updateSwapRouter,
+} from './adminTasks';
+
+/**
+ * Updates the values of the batch fees of the BatchPayments contract, if needed
+ * @param contractAddress address of the BatchPayments Proxy
+ * @param hre Hardhat runtime environment
+ */
+const setupBatchPayments = async (
```

Comment:
this setupBatchPayments is the same function from setupBatchPayments.ts
You can import that one here right ?